src/frontend/src/routes/(new-styling)/manage/(authenticated)/(access-and-recovery)/recovery/+page.svelte

```diff
@@ -145,6 +171,49 @@
     });
     return true;
   };
+  const handleUnlock = async (recoveryPhrase: string[]): Promise<boolean> => {
```

Comment:
It's not clear from this function's name what is does. 

How about


```suggestion
  const isRecoveryPhraseCorrect = async (recoveryPhrase: string[]): Promise<boolean> => {
```